compute/hpc_backend.go

```diff
@@ -55,6 +55,10 @@ func (b *HPCBackend) WriteEvent(ctx context.Context, ev *events.Event) error {
 	return nil
 }
 
+func (b *HPCBackend) Close() {
+	//TODO: Close things here?
```

Comment:
If really a TODO, add test ...  If a NOOP, add comment to say so

Reply:
Removed comment